libs/application/templates/party-application/src/lib/PartyApplicationTemplate.ts

```diff
@@ -10,8 +10,50 @@ import {
 } from '@island.is/application/core'
 import { dataSchema } from './dataSchema'
 import { assign } from 'xstate'
-import { API_MODULE_ACTIONS, States, Roles } from '../constants'
+import { ApiModuleActions, States, Roles } from '../constants'
+import { EndorsementListTags } from '../constants'
 
+const assignees = (constituency: EndorsementListTags) => {
+  switch (constituency) {
+    case 'partyApplicationReykjavikurkjordaemiSudur2021': {
+      return (
+        process.env.PARTY_APPLICATION_RVK_SOUTH_ASSIGNED_ADMINS?.split(',') ??
```

Comment:
everytime i read these, i think this is an application for having a party

should we be reading these directly from env variables? would it be better to put them in the environments config file, where they are read with other env variables?